packages/rum/src/domain/getSessionReplayLink.ts

```diff
@@ -0,0 +1,50 @@
+import type { RumConfiguration, RumSessionManager, ViewContexts } from '@datadog/browser-rum-core'
+import { getDatadogOrigin } from '@datadog/browser-rum-core'
+import { isBrowserSupported } from '../boot/isBrowserSupported'
+import { getReplayStats } from './replayStats'
+
+export function getSessionReplayLink(
+  configuration: RumConfiguration,
+  sessionManager: RumSessionManager,
+  viewContexts: ViewContexts
+): string | undefined {
+  const session = sessionManager.findTrackedSession()
+  const queryParams: Record<string, string> = {}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Similarly to [`endpointBuilder`](https://github.com/DataDog/browser-sdk/blob/c0cd6b589368856db5c43d1aa91cd2a99618220c/packages/core/src/domain/configuration/endpointBuilder.ts#L114), using a string array would be simpler than a record here:

```ts
const parameters = []
...
parameters.push(`seed=${view.id}`)
...
parameters.join('&')
```